cluster-endpoints/src/grpc_subscription.rs

```diff
@@ -411,23 +414,46 @@ pub fn create_grpc_subscription(
         create_grpc_multiplex_processed_slots_subscription(grpc_sources.clone());
 
     let (block_multiplex_channel, jh_multiplex_blockstream) =
-        create_grpc_multiplex_blocks_subscription(grpc_sources);
+        create_grpc_multiplex_blocks_subscription(grpc_sources.clone());
 
     let cluster_info_polling = poll_cluster_info(rpc_client.clone(), cluster_info_sx);
     let vote_accounts_polling = poll_vote_accounts(rpc_client.clone(), va_sx);
 
-    let streamers = EndpointStreaming {
-        blocks_notifier: block_multiplex_channel,
-        slot_notifier: slot_multiplex_channel,
-        cluster_info_notifier,
-        vote_account_notifier,
-    };
-
-    let endpoint_tasks = vec![
-        jh_multiplex_slotstream,
-        jh_multiplex_blockstream,
-        cluster_info_polling,
-        vote_accounts_polling,
-    ];
-    Ok((streamers, endpoint_tasks))
+    // accounts
+    if !accounts_filter.is_empty() {
```

Comment:
is that the feature activation condition? that's quite implicit.

Reply:
Yes it is.